blueocean-core-js/src/less/run-button.less

```diff
@@ -11,6 +11,18 @@
         cursor: pointer;
     }
 
+    .run-button {
+        &.btn-secondary {
+            svg {
+                fill: #4A90E2;
+            }
+        }
+
+        svg {
+            fill: #ffffff;
+        }
+    }
+
```

Comment:
Can we integrate this with the `.action-button-colors` mixin? It's used just below this. Ideally we can re-use that mixin with the more-specific selector, or if there's a problem with the mixin we might want to update it? It's also defined in dashboard.

Ideally we want to get away from using hex codes in individual-component `.less` files, and reuse (or define new) colour/style names in `variables.less`

Reply:
This stuff is going to need some overhaul in the next couple of months with the need for theming coming from Blue Steel, so the fewer places we need to look for codes, the easier that process will go.

Reply:
In general, it would be simple to just override styles with a downstream CSS file, or build a custom one from the original. Don't think it's necessary to allow custom colors and styles etc. to be passed in. Not sure exactly what you're saying, we might be talking about exactly the same thing :)